phone/app/src/main/java/jp/ac/mayoi/maigocompass/bottomNavigation/BottomNavigationItem.kt

```diff
@@ -0,0 +1,61 @@
+package jp.ac.mayoi.maigocompass.bottomNavigation
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import jp.ac.mayoi.core.resource.colorAccent
+import jp.ac.mayoi.core.resource.colorTextCaption
+import jp.ac.mayoi.core.resource.textStyleCaption
+
+@Composable
+fun BottomNavItem(
+    @DrawableRes resId:Int,
+    label:String,
+    onClick:()->Unit,
+    isSelected:Boolean,
+){
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.clickable { onClick() }
+    ){
+        Icon(
+            painter = painterResource(resId),
+            tint = if(isSelected){
+                colorAccent
+            }else{
+                colorTextCaption
+            },
+            contentDescription = null,
+            modifier = Modifier.size(40.dp),
+        )
+        Text(
+            text = label,
+            style = textStyleCaption,
+            color = if(isSelected){
+                colorAccent
+            }else{
+                colorTextCaption
+            },
+        )
+    }
+}
+
+@Preview
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/fd31e7f306bbb6cb35174ceecb2a983d64de68e0/phone/app/src/main/java/jp/ac/mayoi/maigocompass/bottomNavigation/BottomNavigationItem.kt#L52"></span>Composables annotated with @Preview that are used only for previewing the UI should not be public.See https://slackhq.github.io/compose-lints/rules/#preview-composables-should-not-be-public for more information.</td>
    </tr>
  </tbody>
</table>